src/main/resources/static/js/switch-instances.js

```diff
@@ -1,6 +1,5 @@
 jQuery(document).ready(function () {
     var frontendServiceUrl = $('#frontendServiceUrl').text();
-    var frontendServiceBackEndPath = "/backend";
     document.getElementById('switcher').disabled = true;
```

Comment:
" instead of single quotation

Reply:
Line 2 should not be needed since it is a global variable, will remove in next commit.

Reply:
Done.